crates/rolldown_resolver/Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository.workspace = true
 workspace = true
 
 [dependencies]
-oxc_resolver   = { workspace = true }
+oxc_resolver   = { workspace = true, features = ['package_json_raw_json_api'] }
```

Comment:
What's needed from `package_json_raw_json_api`? Let's try and avoid this for performance reasons.

Reply:
I upgrade the version, already fix it.